src/MercadoPago/Client/Order/OrderTransactionClient.cs

```diff
@@ -0,0 +1,197 @@
+﻿namespace MercadoPago.Client.Order
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using MercadoPago.Error;
+    using MercadoPago.Http;
+    using MercadoPago.Resource.Order;
+    using MercadoPago.Serialization;
+
+    /// <summary>
+    /// Client that use the Order APIs.
+    /// </summary>
+    public class OrderTransactionClient : MercadoPagoClient<OrderTransaction>
+    {
+
+        private readonly OrderTransactionUpdateClient transactionUpdateClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrderTransactionClient"/> class.
+        /// </summary>
+        /// <param name="httpClient">The http client that will be used in HTTP requests.</param>
+        /// <param name="serializer">
+        /// The serializer that will be used to serialize the HTTP requests content
+        /// and to deserialize the HTTP response content.
+        /// </param>
+        public OrderTransactionClient(IHttpClient httpClient, ISerializer serializer)
+            : base(httpClient, serializer)
+        {
+            transactionUpdateClient = new OrderTransactionUpdateClient(httpClient, serializer);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrderTransactionClient"/> class.
+        /// </summary>
+        /// <param name="httpClient">The http client that will be used in HTTP requests.</param>
+        public OrderTransactionClient(IHttpClient httpClient)
+            : base(httpClient, null)
+        {
+        }
```

Comment:
[req] Função vazia, faltou implementar a instanciação do Transaction Client
```suggestion
        public OrderTransactionClient(IHttpClient httpClient)
            : base(httpClient, null)
        {
        }
```

Reply:
Não precisa, vide demais implementações de client.